manifests/install.pp

```diff
@@ -80,6 +80,12 @@
       ensure => "${package_version}-1.el${::operatingsystemmajrelease}",
       *      => $_package_options,
     }
+  } elsif ($::osfamily == 'Debian') and ($package_version != 'present') {
+    # Workaround PUP-5802/PUP-5025
```

Comment:
Oh actually, is there a ticket for debian platforms that is similar to this ticket?